css/css-flexbox/percentage-max-height-004-ref.html

```diff
@@ -0,0 +1,3 @@
+<!-- quirks mode -->
+<div style="width:100px; height:100px; background:green;"></div>
+
```

Comment:
Perhaps we can move this test to the reference/ folder.

Reply:
maybe we can reuse the ref file used in the other percentage-max-height tests.

Reply:
The test is in quirk-mode. Inserting text in the test script introduces a scrolling bar which makes it difficult to share the same ref as other percentage-max-height tests.